101-cosmosdb-create-arm-template/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "apiType": {
```

Comment:
remove the non-gen values and just use defaultValues in the template... for the locations use [resourceGroup().location] rather than hardcoding a location

Reply:
I'm not sure what you mean by using "defaultValues" -- I have updated the location values.

Reply:
From the parameter file and the template file, remove both writeRegionLocation and readRegionLocation parameters.
In the template, at the places where refer to these two parameters, hardcode the value as:
"somelocation"="[resourceGroup().location]",